perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -2996,43 +3043,48 @@ public void setPerunBl(PerunBl perunBl) {
 	}
 
 	/**
-	 * Creates a user account in external system if does not exist yet
+	 * Handles sponsorship for the given user using the given userData. This usually means that
+	 * an account in an external system is created, if needed.
```

Comment:
Please describe all use-cases, like when user already have login in specified namespace, input is ignored and no change is done to the users login/password.